gentoo-i686/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM gentoo/stage3-x86
+LABEL MAINTAINER="OSU Open Source Lab <dockerhub@osuosl.org>"
+RUN echo 'GENTOO_MIRRORS="http://gentoo.osuosl.org"' >> /etc/portage/make.conf && \
```

Comment:
not `https`?

Reply:
Suppose I could change it to that :)

Reply:
@ramereth Wait, why was this not implemented? Does gentoo not have the correct CA certs at this point?